client/Dockerfile.fedora

```diff
@@ -3,7 +3,7 @@ FROM osbs-box:fedora
 RUN dnf -y --refresh install \
       origin-clients \
       iproute \
-      koji-containerbuild-cli \
+      koji-containerbuild \
```

Comment:
Why not just cli?

Reply:
The package doesn't exist in F29

Reply:
You're right. I believe it changed at some point. It's now only `python2-koji-containerbuild-cli`. Installing all of `koji-containerbuild` is also probably fine.